src/example.test.cpp

```diff
@@ -1,18 +1,15 @@
+#include <iostream>
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
-#include <iostream>
 
 class ClassToTest {
-   public:
-   virtual void functionToMock() {
-   }
-   virtual void callMockedFunction() {
-      functionToMock();
-   }
+  public:
+   virtual void functionToMock() {}
+   virtual void callMockedFunction() { functionToMock(); }
```

Comment:
Maybe not so important here, but do we really want to allow method declaration and implementation on the same line?

Reply:
In general, yes. But we want to have the declaration in the header files and the implementation in the cpp files, where this is done automatically. Thus this should not be an issue.
The example.test.cpp can be deleted later on, when we dont need it anymore for easy reference.